configuration-example.yaml

```diff
@@ -44,40 +44,19 @@ authentication:
     password: password
 
 api:
-  endpointUri: https://api.oregonstate.edu/v1/web-api-skeleton/
+  endpointUri: https://api.oregonstate.edu/v1/
 
-#
-## Example Oracle configuration
-#
-#database:
-#  driverClass: oracle.jdbc.OracleDriver
-#  url: jdbc:oracle:thin:@(DESCRIPTION=(LOAD_BALANCE=ON) (ADDRESS=(PROTOCOL=TCP)(Host=db1.example.com)(Port=1521)) (ADDRESS=(PROTOCOL=TCP)(Host=db2.example.com)(Port=1521)) (CONNECT_DATA=(SERVICE_NAME=example.db)))
-#  user: Username
-#  password: Password
-#  abandonWhenPercentageFull: 0
-#  alternateUsernamesAllowed: false
-#  commitOnReturn: false
-#  autoCommitByDefault: true
-#  readOnlyByDefault: false
-#  properties:
-#    hibernate.dialect: org.hibernate.dialect.Oracle10gDialect
-#  defaultCatalog: # none
-#  defaultTransactionIsolation: default
-#  useFairQueue: true
-#  initialSize: 10
-#  minSize: 10
-#  maxSize: 20
-#  initializationQuery: # none
-#  logAbandonedConnections: true
-#  logValidationErrors: true
-#  maxConnectionAge: # none
-#  maxWaitForConnection: 1 # second
-#  minIdleTime: 1 # minute
-#  validationQuery: "SELECT 1 FROM dual"
-#  validationQueryTimeout: # none
-#  checkConnectionWhileIdle: true
-#  checkConnectionOnConnect: false
-#  checkConnectionOnReturn: false
-#  autoCommentsEnabled: true
-#  evictionInterval: 5 # seconds
-#  validationInterval: 30 # seconds
+httpClient:
+  timeout: 150000ms
+  connectionTimeout: 15000ms
+  timeToLive: 1h
+  cookiesEnabled: false
+  maxConnections: 2
+  maxConnectionsPerRoute: 2
+  keepAlive: 0ms
+  retries: 0
+  userAgent: beaver-bus-api
+
+rideSystems:
```

Comment:
BeaverBusConfiguration is looking for `ridesystems` but this is using `rideSystems`. I'm good with either one being changed, but it probably makes more sense for it to be camel case.

https://github.com/osu-mist/beaver-bus-api/blob/11a6a3a3b964754be1564a1b615e31fce1a77dac/src/main/groovy/edu/oregonstate/mist/beaverbus/BeaverBusConfiguration.groovy#L10

Reply:
Ugh, thanks for catching that. It's very easy for the example configuration to diverge with the actual configuration.